.github/workflows/build-package-docs.yaml

```diff
@@ -140,8 +146,10 @@ jobs:
              -d '{"msgtype": "m.text", "body": "${{ env.FAIL_MESSAGE }}"}'
 
   deploy-package-docs:
-    if: fromJSON(github.event.inputs.deploy)
```

Comment:
Can't you just change this to `if: github.event.inputs.deploy` (without the `fromJSON`) and keep the rest as is?

Reply:
Unfortunately, no. If it's not wrapped in `fromJSON` then the "deploy-package-docs" job always runs when triggered by a workflow dispatch, even if you don't select the "deploy" checkbox.

Here are two runs from my fork. The first I did not select "deploy" and the second I did. In both, the deploy job was attempted (although it failed for other reasons).

- https://github.com/oraNod/ansible-documentation/actions/runs/10573538948
- https://github.com/oraNod/ansible-documentation/actions/runs/10573540555

When triggered by a schedule, if the `fromJSON` is omitted, you get the same error that I mentioned in the PR description.

```
The template is not valid. Newtonsoft.Json.JsonReaderException: Error reading JToken from JsonReader.
```

Here is a scheduled run from my fork: https://github.com/oraNod/ansible-documentation/actions/runs/10573983190